modules/kueezRtbBidAdapter.js

```diff
@@ -14,40 +15,63 @@ const BIDDER_CODE = 'kueezrtb';
 const BIDDER_VERSION = '1.0.0';
 export const storage = getStorageManager({bidderCode: BIDDER_CODE});
 
+export const spec = {
+  code: BIDDER_CODE,
+  version: BIDDER_VERSION,
+  gvlid: GVLID,
+  supportedMediaTypes: [BANNER, VIDEO],
+  isBidRequestValid,
+  buildRequests: createBuildRequestsFn(createDomain, createUniqueRequestData, storage, BIDDER_CODE, BIDDER_VERSION, false),
+  interpretResponse: createInterpretResponseFn(BIDDER_CODE, false),
+  getUserSyncs: createUserSyncGetter({
+    iframeSyncUrl: 'https://sync.kueezrtb.com/api/sync/iframe',
+    imageSyncUrl: 'https://sync.kueezrtb.com/api/sync/image'
+  }),
+  createFirstPartyData,
+};
+
 export function createDomain(subDomain = DEFAULT_SUB_DOMAIN) {
   return `https://${subDomain}.kueezrtb.com`;
 }
 
-function createUniqueRequestData(hashUrl, bid) {
-  const {
-    auctionId,
-    transactionId,
-  } = bid;
+export function getAndSetFirstPartyData() {
+  if (!storage.hasLocalStorage()) {
+    return;
+  }
+  let fdata = tryParseJSON(storage.getDataFromLocalStorage('_iiq_fdata'));
+  if (!fdata) {
+    fdata = spec.createFirstPartyData();
+    storage.setDataInLocalStorage('_iiq_fdata', JSON.stringify(fdata));
```

Comment:
Sonobi Bid Adapter and IntentIq use this same key name, this will result in your adapter's data overwriting theirs. Do you want to put this in a more specific key name?

Reply:
I only write if it does not exist.
No need for another key name.